lib/openhab/dsl/items/timed_command.rb

```diff
@@ -143,6 +143,25 @@ def command(command, for: nil, on_expire: nil, &block)
           self
         end
 
+        #
+        # @return [true, false] true if a timed command is active for the item
+        #
+        # @example
+        #   rule "Motion Detected" do
+        #     received_command Motion_Sensor, command: ON
+        #     run do
+        #       # Do not activate timer if the light was already turned on manually
+        #       next if FrontPorchLight.on? && !FrontPorchLight.timed_command?
+        #
+        #       # The timer extends when more motion is detected
+        #       FrontPorchLight.ensure.on for: 30.minutes
+        #     end
+        #   end
+        #
+        def timed_command?
```

Comment:
so my only worry with this is the interface is the same as "timer running?", which is possibly open to race conditions. can you think a bit on how you could extend this (possibly with an optional block?) to give some amount of the functionality you get with TimerManager and rescheduling timers that may or may not be active?

Reply:
race conditions are tricky stuff! I haven't thought about how it can happen in this situation.

In regards to your suggestion, are you thinking about a block for a `#timed_command` method, similar to `TimerManager#schedule` instead of (i.e. remove) `#timed_command?`

Reply:
Isn't this the same as `timers.include?(timer_id)`?

Reply:
Yes. which includes a nice big caveat about it possibly introducing race conditions. I'm okay with this as-is if you add a similar caveat.

Reply:
I pushed a separate commit, removing #timed_command? and using an extra parameter for timed command instead. If you think this new approach is OK, I'll squash and change the commit message / PR title accordingly.

Reply:
It feels that this is for such a very specific use case, not as generic as TimerManager#schedule. But I'm not sure if a generic solution like that is needed. Maybe it is, but I don't have a picture of what it should look like based on what I currently need. Open for ideas!